war/src/main/java/executable/Main.java

```diff
@@ -369,17 +360,6 @@ public static void main(String[] args) throws IllegalAccessException {
         }
     }
 
-    @SuppressFBWarnings(value = "DM_DEFAULT_ENCODING", justification = "--logfile relies on the default encoding, fine")
-    private static PrintStream createLogFileStream(File file) {
-        LogFileOutputStream los;
-        try {
-            los = new LogFileOutputStream(file);
-        } catch (FileNotFoundException e) {
-            throw new UncheckedIOException(e);
-        }
-        return new PrintStream(los);
```

Comment:
This is now handled by the call to `new PrintStream` in Winstone's `Launcher#initLogger`. Critically, we preserve compatibility by keeping the (implicit here, explicit in Winstone) default character set.